gr-soapy/python/soapy/bindings/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ include(GrPybind)
 # Python Bindings
 ########################################################################
 
+# For setting string conversions
```

Comment:
Not sure, but it seems like this is referencing an include file in a lib (private) dir as part of the public API?

Reply:
This points to the string <-> setting conversion code we went back and forth on, which this commit moves to the lib directory to be used there too.